src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -361,7 +382,7 @@ final class OnboardingInfoViewController: UIViewController {
 				persistForDPP(accepted: false)
 				completion?()
 			} else {
-				self.exposureManagerActivated = true
+				self.store.exposureActivationOnboardingState = .activated
```

Comment:
Hmm why not keep it as an instance variable but store it the store? 🤔
Keeping it as an instance variable would reset the flag after a restart of the app which would be desirable if ENF is acting up.

Reply:
ENF gets activated while onboarding gets show. But if you navigate back the view controller gets freed from memory. Navigating forward again will create a new instance, not knowing that ENF was activated already.
Maybe we need to exclude the flag from reset logic? But the behavior isn't different than before.

Reply:
As discussed, I found a better solution. Please have a 2nd look.